docs/modules/gear/actions-triggers.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: actions-triggers
+title: Actions & Triggers
+---
+
+Actions are a set of features that are applied to players, teams or matches, similiar to [Kits](kits).
+
+##### Action Elements
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Element</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`<action> </action>`}</label>
+        </td>
+        <td>A single action.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`<trigger/>`}</label>
+        </td>
+        <td>Creates a trigger to cause an Action.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`<message/>`}</label>
+        </td>
+        <td>A message that is sent to the player.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`<switch-scope> </switch-scope>`}</label>
+        </td>
+        <td>Changes a scope in which the trigger looks at.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`<kit/>`}</label>
+        </td>
+        <td>
+          Applies a <a href="#kits">Kit</a>.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+##### Action Attributes
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Attribute</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`id`}</label>
+        </td>
+        <td>An ID.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`scope`}</label>
+        </td>
+        <td>
+          Runs the filter against, <label>{`player`}</label>, <label>{`team`}</label>
+          , or
+          <label>{`match`}</label>.
+        </td>
+      </tr>
```

Comment:
also has a filter attribute, which will get tested before running the children actions inside (part of variables PR, not *yet* merged but should be soon enough). The filter here does not need to be dynamic